src/detections/detection.rs

```diff
@@ -41,29 +46,59 @@ impl Detection {
                     match quick_xml::de::from_str(&r.data) {
                         Ok(event) => {
                             let event: event::Evtx = event;
-
                             let event_id = event.system.event_id.to_string();
                             let channel = event.system.channel.to_string();
                             let event_data = event.parse_event_data();
 
                             &common.detection(&event.system, &event_data);
                             if channel == "Security" {
-                                &security.detection(
-                                    event_id,
-                                    &event.system,
-                                    &event.user_data,
-                                    event_data,
-                                );
+                                match event_id.as_str() {
+                                    "7030" | "7036" | "7045" | "7040" | "104" => {
+                                        &security.detection(
+                                            event_id,
+                                            &event.system,
+                                            &event.user_data,
+                                            event_data,
+                                        );
+                                    }
+                                    _ => println!("Not Match is Event ID Num:{}", event_id),
+                                }
                             } else if channel == "System" {
-                                &system.detection(event_id, &event.system, event_data);
+                                match event_id.as_str() {
+                                    "4688" | "4672" | "4720" | "4728" | "4732" | "4756"
```

Comment:
channelの指定がSystemとなっているが指定しているeventIDがSecurityになっています。
L68-69の条件を以下のように変更していただくと正しいと思いますがいかがでしょうか。

```suggestion
                                    "7030" | "7036" | "7045" | "7040" | "104"
```

Reply:
46022cc により修正をしました。